nautobot/extras/forms/mixins.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, *args, **kwargs):
                     set_initial=False, enforce_required=False, simple_json_filter=True
                 )
             else:
-                self.fields[field_name] = cf.to_form_field(set_initial=False, enforce_required=False)
+                self.fields[field_name] = cf.to_filter_field(set_initial=False, enforce_required=False)
```

Comment:
Why using `to_filter_field` inside of a form mixin? This seems a little confusing.